module/VuFind/src/VuFind/Controller/InstallController.php

```diff
@@ -580,7 +581,8 @@ protected function checkILS()
             $status = false;
         } else {
             try {
-                $status = 'ils-offline' !== $this->getILS()->getOfflineMode(true);
+                $status = 'ils-offline' !== $this->getILS()->getOfflineMode(true)
+                    || ($this->getILS()->getDriver() instanceof NoILS);
```

Comment:
I think it might be better to read the setting from config.ini instead of fetching the driver object. Otherwise, if NoILS failover is enabled and the primary driver is having problems, this will suppress warnings about the issue.

Reply:
Right, of course. Fixed.